I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scenes/script.rpy", line 13, in script call
    call intro from _call_intro
  File "game/scenes/act1a.rpy", line 282, in script
    jump act2a
ScriptError: could not find label 'act2a'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scenes/script.rpy", line 13, in script call
    call intro from _call_intro
  File "game/scenes/act1a.rpy", line 282, in script
    jump act2a
  File "C:\Users\kasey\renpy-8.1.1-sdk\renpy\ast.py", line 1992, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\kasey\renpy-8.1.1-sdk\renpy\script.py", line 1013, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'act2a'.

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.1.23060707
Snake Bitten 1.0
Fri Oct 20 22:40:07 2023
